woocommerce/woocommerce
# Update JS packages changelogs

### All Submissions:






### Changes proposed in this Pull Request:

See https://github.com/woocommerce/woocommerce/pull/33359 for an example of this process for a single package.

* New Changelog file generated with new format
* Update changelog and package.json to prep release
* Include a `.gitignore` so that `pnpm publish` won't bundle up things like `vendor/`

Part of https://github.com/woocommerce/woocommerce/issues/33317

### How to test the changes in this Pull Request:

1. Spot check that things make sense, like version numbers changelog formatting etc. It may be easier to pull this down and check in your IDE.

Note that private packages were not changed. Public packages that Solaris owns were not changed as well. Those will be brought up to speed once all automation and tooling is in place.

**Note** Changelogger validation fails when it should not. Here is the issue for that https://github.com/woocommerce/woocommerce/issues/33424

### Other information:




### FOR PR REVIEWER ONLY:
